source/02-EV/05.ptx

```diff
@@ -10,6 +10,39 @@
     </objectives>
 
 
+    <subsection>
+        <title>Warm Up</title>
+        <activity>
+            <introduction>
+                <p>
+                    Consider the following set of ingredients:
+                    <me>S=\left\{\textrm{tomato}, \textrm{olive oil}, \textrm{dough}, \textrm{cheese}, \textrm{pizza sauce}, \textrm{garlic}\right\}</me>
+                </p>
+            </introduction>
+            <task>
+                <statement>
+                    <p>
+                        Does "pizza" live inside of <m>\vspan(S)</m>?
+                    </p>
+                </statement>
+            </task>
+            <task>
+                <statement>
+                    <p>
+                        Would you say that the set <m>S</m> is linearly independent or dependent?
+                    </p>
+                </statement>
+            </task>
```

Comment:
```suggestion
            <task>
                <statement>
                    <p>
                        Identify some ingredients in <m>S</m> that make the set linearly dependent.
                    </p>
                </statement>
            </task>
```

Fixes the issue that the question right now just prompts students to say "independent" or "dependent", and that the answer "dependent" is given away in the next task.